### ISSUE TITLE
Django 1.10 Deprecation Warning: SubFieldBase

### ISSUE THREAD

**Issue by coredumperror** [torchbox/wagtail#2295]
Using Django 1.9 and Wagtail 1.3.1, I see this warning issued whenever I execute any manage.py command:

```
/Users/rrollins/.pyenv/versions/wagtail-bootstrap/lib/python2.7/site-packages/django/utils/six.py:808: RemovedInDjango110Warning: SubfieldBase has been deprecated. Use Field.from_db_value instead.
  return meta(name, bases, d)
```

The offending code is on line 44 in wagtailcore/fields.py:
```
class StreamField(with_metaclass(models.SubfieldBase, models.Field)):
```

I'd make this a PR, but I don't know the appropriate fix. [Here's](https://docs.djangoproject.com/en/1.8/releases/1.8/#subfieldbase) the description of the deprecation in the Django changelog. Unfortunately, I'm not well enough versed in the low-level details of Django's model system to understand how to convert those instructions into whatever code changes may be needed in `StreamField`. Another django plugin's [fix](https://github.com/contracode/python-social-auth/commit/10a6f5570a4471d156c2bc06ecd4313189d23172) was simply removing the `with_metaclass(models.SubfieldBase, ` part, but I'm not sure that would be sufficient or appropriate here.

**Comment by tomdyson** [comment]
Thanks @coredumperror. Django 1.10 isn't scheduled until August - https://code.djangoproject.com/wiki/Version1.10Roadmap - so we have a little time to fix this!

**Comment by coredumperror** [comment]
True, but it's pretty annoying that all my cron jobs need special configuration so as not to email me about this warning every time they run.

**Comment by tomdyson** [comment]
If you don't want to adjust your cron jobs you can add a logging filter to your settings.py: http://stackoverflow.com/a/31103483/181793

**Comment by tomdyson** [comment]
In general I agree: we should seek to eliminate deprecation warnings. They're a nuisance for cron jobs and alarming for newcomers.

**Comment by coredumperror** [comment]
Oh that is *slick*! Thanks for pointing me to that.

**Comment by gasman** [comment]
Already reported at #2070, fix in progress at #2119.

**Comment by coredumperror** [comment]
Ah dangit! I thought my search for "SubFieldBase" would have been sufficient, but it didn't find that PR for some reason. I'm not very well versed with Github's search function, though, so I probably did it wrong.
